vm-setup/roles/v1aX_integration_test/tasks/verify.yml

```diff
@@ -88,6 +88,15 @@
           replace: "{{ POD_CIDR }}"
         register: updated_manifest
 
+      - name: Add IP_AUTODETECTION_METHOD in calico config
+        blockinfile:
+          path: /tmp/calico.yaml
+          insertafter: "192.168.0.0/18"  
```

Comment:
maybe we should use `POD_CIDR` variable instead of explicitly using the IP?

Reply:
Done